### PR TITLE
REL: v1.0.0

### PR DESCRIPTION
**Pull request by dmgav**

### CODE REVIEW

**Comment by dmgav** [comment]
I created PR to update `lmfit` to v1.0.0.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,12 +20,12 @@ requirements:
...
-    - asteval>=0.9.11
-    - numpy
-    - scipy>=0.17
+    - python>=3.5
+    - asteval>=0.9.16
+    - numpy>=1.16
+    - scipy>=1.2
     - six
-    - uncertainties>=3.0
+    - uncertainties>=3.0.1
```
Comment (mrakitin) [inline recipe/meta.yaml]: Conda-forge community recommends to use a blank space between the package name and the comparisson with version (only before >, <, =, not after). Could you please fix the recipe?

Comment (dmgav) [reply recipe/meta.yaml]: I actually followed the pattern. The version numbers are the minimal versions from Lmfit documentation.

Comment (mrakitin) [reply recipe/meta.yaml]: The pattern was originally wrong :)

**Pull request merged**